### ISSUE TITLE
Plugin incompatibility with IntelliJ IDEA 2023.3.6 Community

### ISSUE THREAD

**Issue by ketura** [Zelaux/HjsonSupportPlugin#2]
I'm receiving the following error upon upgrading IDEA Community edition to 2023.3.6:

![image](https://github.com/Zelaux/HjsonSupportPlugin/assets/1814132/3d8eb995-48dc-45cc-a817-f1670cbfd29e)

Based on the error, it looks like the version format may have been altered, and any compatibility settings may need updated to match.

**Comment by ketura** [comment]
Indeed, it looks like the build format changed: https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html#build-number-format

> To denote a release, a multipart build number is used. It consists of the following parts:
> * Product ID (IC for IDEA Community, IU for IDEA Ultimate, RM for RubyMine, PY for PyCharm, PS for PhpStorm, etc.)
> * Branch number (223)
> * Build number in the branch (9559)

So the plugin.xml needs updating to include the product ID.

**Comment by ketura** [comment]
Oh wait, it's because IntelliJ Idea is now up to 233, duh.  So the build.gradle.kts needs updated at least.

**Comment by SebastianSolidwork** [comment]
I got similar problems with the Ultimate version.
"Not compatible with the version of your running IDE (IntelliJ IDEA 2023.3.6)" from my https://plugins.jetbrains.com/plugin/22501-hjson
The compatibility range is exceed : 2022.2 — 2023.2.7

**Comment by Xaekai** [comment]
Any chance of an update on this?

**Comment by samogot** [comment]
Same problem. Is there any incompatible API, or it is just the matter of bumping the version in the configuration and rebuilding?

**Comment by ketura** [comment]
> Same problem. Is there any incompatible API, or it is just the matter of bumping the version in the configuration and rebuilding?

I'm pretty sure it's just the version needing incremented (or rather, the version definition being reworked to match the new numbering scheme that Intellij now follows). And if it's not, that step has to happen to even run the thing and see if there's other incompatibilities.
